include/fizzy/fizzy.h

```diff
@@ -1,6 +1,8 @@
-// Fizzy: A fast WebAssembly interpreter
-// Copyright 2020 The Fizzy Authors.
-// SPDX-License-Identifier: Apache-2.0
+/// @file
```

Comment:
Without this file is considered private by default and documentation for free functions is not generated.

Reply:
Are you sure it can't just be added in the doxygen config?

Reply:
Well adding the file explicitly in `INPUT` doesn't help (it's included anyway)

The only other way to fix it that I found is to set `EXTRACT_ALL = YES`, but it might include more stuff than needed, EVMC has it at `NO`.

Reply:
https://www.doxygen.nl/manual/commands.html#cmdfile

> The documentation of global functions, variables, typedefs, and enums will only be included in the output if the file they are in is documented as well or if EXTRACT_ALL is set to YES.

Reply:
This is done wrongly. We don't want to the "copyright header" to be included in the documentation text.

It rather should be
```
// Copyright...

/// Fizzy public C API.
/// @file
```

Reply:
Changed. I just hate doxygen :angry:

Reply:
`clang-doc` maybe if you want to contribute to LLVM.